### DISCUSSION TITLE
release

### DISCUSSION THREAD

**Discussion by danieleisenhardt** [wovalle/fireorm#271] [General]
Hi,

When will a new version be released? I would like to deploy my own project with the changes in my recently accepted PR.

Thanks.

**Comment by wovalle** [comment]
Oh, I didn't see it didn't got released last time. I'll check what's wrong today

**Reply by danieleisenhardt** [reply]
Did you find out what the problem is?

**Reply by wovalle** [reply]
Sorry it took that long, it should be deployed now.

**Reply by danieleisenhardt** [reply]
No problem, I managed to put my own (ducttape) version on npm in the meantime. I will deprecate that now before more unfortunate people download it.
